### ISSUE TITLE
resume training problem

### ISSUE THREAD

**Issue by Atom0808** [yassouali/pytorch-segmentation#133]
I set the number of iterations to 80 times. I have already trained 40 times. Now I want to continue training, but the program does not start training. Why is this?

<img width="735" alt="11111" src="https://user-images.githubusercontent.com/75409500/147064801-f94f9446-c3a6-4346-955b-919ca224deae.png">
{
    "name": "PSPNet",
    "n_gpu": 1,
    "use_synch_bn": false,

    "arch": {
        "type": "PSPNet",
        "args": {
            "backbone": "resnet50",
            "freeze_bn": false,
            "freeze_backbone": false
        }
    },

    "train_loader": {
        "type": "VOC",
        "args":{
            "data_dir": "data/VOCtrainval_11-May-2012",
            "batch_size": 4,
            "base_size": 400,
            "crop_size": 380,
            "augment": true,
            "shuffle": true,
            "scale": true,
            "flip": true,
            "rotate": true,
            "blur": false,
            "split": "train_aug",
            "num_workers": 0
        }
    },

    "val_loader": {
        "type": "VOC",
        "args":{
            "data_dir": "data/VOCtrainval_11-May-2012",
            "batch_size": 4,
            "crop_size": 480,
            "val": true,
            "split": "val",
            "num_workers": 0
        }
    },

    "optimizer": {
        "type": "SGD",
        "differential_lr": true,
        "args":{
            "lr": 0.01,
            "weight_decay": 1e-4,
            "momentum": 0.9
        }
    },

    "loss": "CrossEntropyLoss2d",
    "ignore_index": 255,
    "lr_scheduler": {
        "type": "Poly",
        "args": {}
    },

    "trainer": {
        "epochs": 80,
        "save_dir": "saved/",
        "save_period": 10,
  
        "monitor": "max Mean_IoU",
        "early_stop": 10,
        
        "tensorboard": true,
        "log_dir": "saved/runs",
        "log_per_iter": 20,

        "val": true,
        "val_per_epochs": 2
    }
}

**Comment by yassouali** [comment]
Hi, i think in the original config, you had the number of epochs equal to 40, so when the config is relaoded, the epochs become 40 instead of 80. 

You can just not load the config file by commenting this line: [here](https://github.com/yassouali/pytorch-segmentation/blob/0da7bd1cd8fec26f2bbbbe5b79641b22e81ff3bc/train.py#L57)
